src/Eccube/Resource/template/admin/Store/plugin_confirm.twig

```diff
@@ -57,7 +57,29 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
                             </div><!-- .plugin-block -->
                         </div><!-- .col-xs-12 -->
                     </div><!-- .row -->
-                </div><!-- /.box-body -->
+                </div><!-- /.box-body 1 -->
+
+                    {% if arrDependency is not empty %}
```

Comment:
arrDependency を利用する意味が無いと思います。
$item->depend に依存情報を持たせるなら、この判断もarrDependency を使う必要ないと思う。
(arrDependency不要なのでは？)

Reply:
@ryo-endo san,
arrDependency is helpful for https://bitbucket.org/lockon_co_jp/eccube_owners-store-repo/pull-requests/12
it can show multi level dependency

Reply:
Considering the lack of information, duplicate plugins, non-existent plugins, so I can not use `$item->depend`. I had to use `$arrDependency`.

Reply:
I see.
検討したうえで必要ということならOKです。了解しました。